Kevin-Muthiani/Arrays_and_Strings/reverse_string_in_place.py

```diff
@@ -0,0 +1,36 @@
+# Write a function reverseSentence(A) that takes in an array of characters, A,
```

Comment:
This works. A cleaner solution would be to convert the input array to string and split by empty spaces, then reverse that array.

Reply:
@einnor I think that would no longer be an 'in-place' solution as required for this assignment since extra memory of would be required to store the string upon joining the input character list and another for storing the words list upon splitting. 
I might however be missing and i have added an out-of-place function for implementing the same. You can review